doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -14,10 +14,7 @@ This tutorial has following sections:
   - [Create starter package](#create-starter-package)
     - [Use Cadl as Input](#use-cadl-as-input)
       - [Create Cadl project](#create-cadl-project)
-      - [Create a sdk project folder](#create-a-sdk-project-folder)
-      - [Generate the library source code](#generate-the-library-source-code)
-      - [Build the library project](#build-the-library-project)
-      - [Export the library's public API](#export-the-librarys-public-api)
+      - [Create sdk starter package](#create-sdk-starter-package)
     - [Use swagger as Input](#use-swagger-as-input)
   - [Add package ship requirements](#add-package-ship-requirements)
     - [Tests](#tests)
```

Comment:
Please correct L35; the version of Visual Studio being referred to is 2022.   There is no version 2020.   Please also remove "VS" and change to "Visual Studio" for clarity.

Reply:
Please add a space to "VisualStudio" on L34.

Reply:
resolved. Thanks

Reply:
This is not ready to be resolved.   Please change "VS" to "Visual Studio" for clarity.